kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/ProxyReconcilerIT.java

```diff
@@ -193,12 +208,12 @@ void testDelete() {
     }
 
     @Test
-    void testUpdateVirtualCluster() {
+    void testUpdateVirtualClusterTargetBootstrap() {
```

Comment:
I added another IT to [56](https://github.com/k-wall/kroxylicious/pull/56) testing that we can update the clusterRef on a virtual cluster.